### ISSUE TITLE
[Feature Request] Compute only "true" delta in calculate delta node

### ISSUE THREAD

**Issue by dmohns** [thingsboard/thingsboard#10224]
**Is your feature request related to a problem? Please describe.**

Currently, the `calculate delta` always computes the delta when an event arrives. It would be great if we can configure the node to only compute "true" deltas. meaning if the delta is 0, then no messages is added to the payload.

The use-case for this is the following: Assume a metering context, where each events encapsulates the current meter reading. When the meter reading changes some derived logic will be executed. Now, if the meter reading changes slower than the cadence of usually recurring events, there will be a lot of 0 delta

**Describe the solution you'd like**

Similar to the `Tell Failure if delta is negative` check box, there could be a new checkbox to `Only compute true delta's`. (The delta output keys will only be added to the message if the difference is > 0. This also applies to time difference if enabled)

**Describe alternatives you've considered**

Currently, we have to add a custom filtering node like so

```
if (msg.hasOwnProperty('meter_reading_delta') && msg['meter_reading_delta'] === 0) {
    delete msg['meter_reading_delta'];
    delete msg['meter_reading_period_ms'];
}
    
return {
    msg: msg,
    metadata: metadata, 
    msgType: msgType
};
```

which works, but is very tedious.

**Comment by ashvayka** [comment]
This is a valid feature request, @ShvaykaD

**Comment by Ultrazombie** [comment]
Hello @dmohns 
Your feature request is merged and will be available in 3.7 of the ThingsBoard version.
https://github.com/thingsboard/thingsboard/pull/10300
